manifest/provider/apply.go

```diff
@@ -299,8 +299,20 @@ func (s *RawProviderServer) ApplyResourceChange(ctx context.Context, req *tfprot
 			rs = c.Resource(gvr)
 		}
 
+		// get fieldManager config
+		fieldManagerName, forceConflicts, err := s.getFieldManagerConfig(plannedStateVal)
+		if err != nil {
+			resp.Diagnostics = append(resp.Diagnostics, &tfprotov5.Diagnostic{
+				Severity: tfprotov5.DiagnosticSeverityError,
+				Summary:  "Could not extract field_manager config",
+				Detail:   err.Error(),
+			})
+			return resp, nil
+		}
+
 		// Check the resource does not exist if this is a create operation
-		if applyPriorState.IsNull() {
+		// forceConflicts will allow creation even if the resource already exists
+		if applyPriorState.IsNull() && !forceConflicts {
```

Comment:
This doesn't appear to be where the check for existence happens, that is done by the `Get()` below.